mcp/mcp-traits/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id("smithy-java.module-conventions")
+    alias(libs.plugins.smithy.gradle.jar)
+}
+
+description = "MCP Traits"
+
+extra["displayName"] = "Smithy :: Java :: MCP Traits"
+extra["moduleName"] = "software.amazon.smithy.java.mcp.traits"
+
+dependencies {
+    api(libs.smithy.model)
+    api(libs.smithy.traitcodegen)
+    implementation(libs.smithy.traitcodegen)
```

Comment:
* `implementation` is redundant with `api`
* Both `api` and `implementation` seem incorrect – did you want `compileOnly`? I wouldn't expect consumers of the generated trait code to need the trait code generator at runtime.